com.unity.ml-agents/Runtime/Academy.cs

```diff
@@ -58,7 +58,7 @@ public class Academy : IDisposable
         /// on each side, although we may allow some flexibility in the future.
         /// This should be incremented whenever a change is made to the communication protocol.
         /// </summary>
-        const string k_ApiVersion = "0.16.0";
+        const string k_ApiVersion = "1.0.0";
```

Comment:
I was planning on waiting until the release branch before setting this to 1.0.0, just in case we need to make other changes between now and then. Up to you though.

Reply:
i can revert it back to 0.16.0

Reply:
for 0.x.y versions, we cannot assume compatibility.  So i'll add that logic in and add a test for it.